### ISSUE TITLE
TidalCycles won't run - Windows 10 64-bit

### ISSUE THREAD

**Issue by abhirajdg** [tidalcycles/atom-tidalcycles#49]
Whenever I try to boot TidalCycles, I get the following message:

': error:
    Could not find module `Sound.Tidal.Context'
    It is not a module in the current program, or in any known package.

 Not in scope: oLatency

 Not in scope: oAddress

 Not in scope: oPort

 Not in scope: cFrameTimespan

    Variable not in scope: streamReplace :: t0 -> t

 Variable not in scope: tidal

    Variable not in scope: streamHush :: t0 -> t

 Variable not in scope: tidal

    * Variable not in scope: streamList :: t0 -> t
    * Perhaps you meant `readList' (imported from Prelude)

 Variable not in scope: tidal

    Variable not in scope: streamMute :: t0 -> t

 Variable not in scope: tidal

    Variable not in scope: streamUnmute :: t0 -> t

 Variable not in scope: tidal

    Variable not in scope: streamSolo :: t0 -> t

 Variable not in scope: tidal

    Variable not in scope: streamUnsolo :: t0 -> t

 Variable not in scope: tidal

    Variable not in scope: streamOnce :: t0 -> t

 Variable not in scope: tidal'

SuperCollider works just fine and Tidal is installed on my machine.

**Comment by yaxu** [comment]
What happens when you do `cabal install tidal` from a terminal window?

**Comment by abhirajdg** [comment]
I removed all dependencies and tried again. This is the output I get on the command line:

cabal install tidal
Warning: The install command is a part of the legacy v1 style of cabal usage.

Please switch to using either the new project style and the new-install
command or the legacy v1-install alias as new-style projects will become the
default in the next version of cabal-install. Please file a bug if you cannot
replicate a working v1- use case with the new-style commands.

For more information, see: https://wiki.haskell.org/Cabal/NewBuild

Resolving dependencies...
Starting     network-3.1.0.1
Failed to install network-3.1.0.1
Build log ( C:\Users\ADG\AppData\Roaming\cabal\logs\ghc-8.6.5\network-3.1.0.1-3hHBpw17wpa9MleAlG8j6i.log ):
Configuring network-3.1.0.1...
configure: WARNING: unrecognized options: --with-compiler
configure: error: cannot run /bin/sh C:\/Users/ADG/AppData/Local/Temp/cabal-tmp-12976/network-3.1.0.1/config.sub
C:\/Users/ADG/AppData/Local/Temp/cabal-tmp-12976/network-3.1.0.1/configure: line 38: sort: No such file or directory
C:\/Users/ADG/AppData/Local/Temp/cabal-tmp-12976/network-3.1.0.1/configure: line 35: sed: No such file or directory
cabal: Leaving directory 'C:\Users\ADG\AppData\Local\Temp\cabal-tmp-12976\network-3.1.0.1'
cabal: Error: some packages failed to install:
hosc-0.17-AeIiOhdUv3P9ZIcUnzxkud depends on hosc-0.17 which failed to install.
network-3.1.0.1-3hHBpw17wpa9MleAlG8j6i failed during the configure step. The
exception was:
ExitFailure 127
tidal-1.0.14-2NMN77JbkRmBxE179tJg3F depends on tidal-1.0.14 which failed to
install.


This wasn't the case the first time around. Problem with the Haskell install here?

**Comment by yaxu** [comment]
Yes, this should solve it
https://forum.toplap.org/t/trouble-launching-tidal-in-atom/678/3

**Comment by abhirajdg** [comment]
And it works! Thanks!
